public/style.css

```diff
@@ -93,6 +98,15 @@ input:focus, select:focus {
   border-color: #333333;
 }
 /*Uploading-photo animation*/
+/* divi? */
+.divi {
```

Comment:
Its on master. andd
I dont use this. will clean the file after merging in nogas and my PRs